src/EditorFeatures/Core/Implementation/Classification/SemanticClassificationBufferTaggerProvider.Tagger.cs

```diff
@@ -99,7 +101,8 @@ private void OnEventSourceChanged(object sender, TaggerEventArgs e)
             {
                 _owner._notificationService.RegisterNotification(
                     OnEventSourceChanged_OnForeground,
-                    _owner._asyncListener.BeginAsyncOperation("SemanticClassificationBufferTaggerProvider"));
+                    _owner._asyncListener.BeginAsyncOperation("SemanticClassificationBufferTaggerProvider"),
+                    _cancellationTokenSource.Token);
```

Comment:
yikes.  That there's  RegisterNotificaiton overload that doesn't take a CT is not good :(